doc/releases/release-notes-2.4.rst

```diff
@@ -311,11 +324,72 @@ Drivers and Sensors
 
 * WiFi
 
+  * Add IPv6 support to Simplelink driver.
+  * Add DNS offloading support to eswifi driver.
+  * Fix esp driver offload protocol parsing.
+  * Fix esp driver GPIO reset control logic.
+  * Fix eswifi driver offloading packet parsing.
 
 
 Networking
 **********
 
+* Enable new TCP stack by default. The legacy TCP stack is not yet removed and
+  can be used if needed.
+* Make network interface a kernel object. This allows better access control
+  handling when usermode is enabled.
+* Use kernel stacks in network related threads, to save memory when usermode
+  is enabled.
+* Enable network statistics collection in key points of the network stack. This
+  can be used to get information where time is spent in RX or TX.
+* The BSD socket sendmsg() can now be used with AF_PACKET type sockets.
+* Add support for enabling OpenThread reference device.
```

Comment:
Not sure if it shall be mentioned in Zephyr release notes, but OpenThread version used with Zephyr was also updated up to upstream commit `ac86fe52e62e60a66aeeb1c905cb1294709147e9`.

Reply:
We have not mentioned these earlier and I wonder how useful it would be for the reader. One can always get the commit id from west.yml file anyway.